randovania/games/samus_returns/assets/hash_words.json

```diff
@@ -0,0 +1,114 @@
+[
+  "Samus",
+  "Metroid",
+  "Alpha",
+  "Gamma",
+  "Zeta",
+  "Omega",
+  "Larva",
+  "Queen",
+  "Baby",
+  "Hatchling",
+  "Arachnus",
+  "Diggernaut",
+  "Proteus",
+  "Ridley",
+  "Animworm",
+  "Autoad",
+  "Autrack",
+  "BigBug01",
+  "BigBug02",
```

Comment:
I dunno if it's that good of an idea to expose internal MSR names like this
(Same with animworm too i think?)

Reply:
I thought it'd be funny tonight include them but yeah those three can go